detox/src/devices/EmulatorDriver.js

```diff
@@ -52,10 +52,18 @@ class EmulatorDriver extends AndroidDriver {
     }
 
     await this._fixEmulatorConfigIniSkinName(name);
-    await this.emulator.boot(name);
```

Comment:
What was the issue here ? boot should return a valid device, either it's shutdown or already booted. If it doesn't we may have issue with `Emulator.js`

Reply:
I now see the comments [here](https://github.com/wix/detox/pull/628#commitcomment-28173251).

Not sure how to proceed here. This fix is in bad taste IMO. 
So the real issue is creating and removing those emulator log files ?

Reply:
I could experiment a bit, I suspect redirecting to different files should fix it, but this was a less user visible change (eg. .gitignore)
Do you not find it logical to only start an emulator if it's not already running?

Reply:
Nope, gave it a try and it still hangs on startup with no output to the log file even if it's a new file each time.
Killing the first emulator then will hang it, killing it causes the second to start up and detox to continue successfully.
Seems like there's something lower level specific to the android emulator going on, possibly needs to have a separate AVD directory?